### DISCUSSION TITLE
Support Angular 16

### DISCUSSION THREAD

**Discussion by aparzi** [assuncaocharles/ngx-indexed-db#366] [Ideas]
You should update the library to support Angular 16

**Comment by kushal-chhetri7** [comment]
True and even Currently the Angular 17 release will have the built-in control flow for templates, a new declarative syntax of writing control flow in the template, thus providing the functionality of *ngIf *ngFor, and *ngSwitch into the angular framework itself.

**Comment by aparzi** [comment]
Hi @kushal-chhetri7,
in this moment opened PR for supporting Angular 16 #367
